clustered/client/src/main/resources/ehcache-clustered-ext.xsd

```diff
@@ -157,13 +158,23 @@
         </xs:annotation>
       </xs:element>
     </xs:sequence>
-    <xs:attribute name="auto-create" type="xs:boolean" use="optional" default="false">
+    <xs:attribute name="auto-create" type="xs:boolean" use="optional">
       <xs:annotation>
-        <xs:documentation> xml:lang="en">
+        <xs:documentation xml:lang="en">
           True if server side components should be automatically created if they are absent.
         </xs:documentation>
       </xs:annotation>
     </xs:attribute>
+    <xs:attribute name="client-mode" type="tc:client-mode-spec" use="optional">
```

Comment:
why don't we set a default to let the user know what's used by default ?

Reply:
See my other reply.